src/fc/fc_domain.c

```diff
@@ -364,6 +399,7 @@ fcConfigAndStartVM(virFCDriver *driver G_GNUC_UNUSED,
     virFCDomainObjPrivate *vmPrivateData = vm->privateData;
     virDomainDiskDef *root_device = NULL;
     g_autofree gchar *computed_cmdline = NULL;
+    size_t i = 0;
```

Comment:
Does this need to be a `size_t`?

Reply:
It has to, there are syntax checks against it

```
>>> MALLOC_PERTURB_=93 /usr/bin/make -C /home/ec2-user/LV/libvirt/build/build-aux sc_prohibit_int_ijk
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
stdout:
make: Entering directory `/home/ec2-user/LV/libvirt/build/build-aux'
prohibit_int_ijk
/home/ec2-user/LV/libvirt/src/fc/fc_domain.c:402:    int i = 0;
make: Leaving directory `/home/ec2-user/LV/libvirt/build/build-aux'
stderr:
build-aux/syntax-check.mk: use size_t, not int/unsigned int for loop vars i, j, k
make: *** [sc_prohibit_int_ijk] Error 1
```